### PR TITLE
Adding mofchecker package

### PR DESCRIPTION
**Pull request by kjappelbaum**
I'm happy to maintain it. 

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/mofchecker/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/mofchecker-{{ version }}.tar.gz
+  sha256: 6c9154d8df22436d3e95594f07deef06f99bbc3d2360f466bda71199e485532f
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: {{ PYTHON }} -m pip install . -vv
```
Comment (dopplershift) [inline recipes/mofchecker/meta.yaml:16]: If you [add the entrypoint](https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python) from the upstream [`setup.py`](https://github.com/kjappelbaum/mofchecker/blob/master/setup.py) you may be able to remove the `skip: true` and make the recipe `noarch: python`.

File: recipes/mofchecker/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click ==7.*
+    - networkx ==2.*
```
Comment (dopplershift) [inline recipes/mofchecker/meta.yaml:24]: Is there any reason these packages are pinned more tightly than [upstream](https://github.com/kjappelbaum/mofchecker/blob/master/requirements.txt).

File: recipes/mofchecker/meta.yaml (hunk 1 above)
Comment (kjappelbaum) [reply recipes/mofchecker/meta.yaml:16]: Thanks for this hint! Just fixed this!

File: recipes/mofchecker/meta.yaml (hunk 2 above)
Comment (kjappelbaum) [reply recipes/mofchecker/meta.yaml:27]: Now aligned the version pins with the `requirements.txt`

**Review by dopplershift — APPROVED** [review]

**Comment by kjappelbaum** [comment]
Thanks @dopplershift!

**Pull request merged**